docs/manual/constant-value-checker.tex

```diff
@@ -251,6 +251,20 @@ \section{Warnings\label{value-checker-warnings}}
 
 \end{itemize}
 \end{sloppypar}
+
+
+\section{Overflow\label{value-checker-overflow}}
+
+The Constant Value Checker normally takes Java's normal overflow rules into account when
+computing the possible values of expressions.
+The \code{-AignoreRangeOverflow} command-line option
+makes it ignore the possibility of overflow
+for range annotations (\<@IntRange>, for instance).
+Ignoring overflow is useful for some checkers
```

Comment:
Sorry I missed this comment before. I've updated the manual. Should I make an entire new changelog entry? It looks like all of the recent ones are releases, but this isn't a release...